index.d.ts

```diff
@@ -69,6 +69,7 @@ declare namespace Dysnomia {
   type GuildTextableChannel = TextChannel | TextVoiceChannel | NewsChannel | StageChannel;
   type GuildTextableWithThreads = GuildTextableChannel | AnyThreadChannel;
   type InviteChannel = InvitePartialChannel | Exclude<AnyGuildChannel, CategoryChannel | AnyThreadChannel>;
+  type PossiblyUncachedInteractionChannel = Textable | { id: string; type: ChannelTypes };
```

Comment:
Maybe use `Pick<Channel, 'id' | 'type'>`?

or use a new interface on line 63?
  ```interface UncachedChannel extends Uncached { type: ChannelTypes }```

Reply:
Textable also doesn't have any channel related properties, is there a better type we can use?

Reply:
Maybe `TextableChannel | PartialChannel` would be a better choice?

Reply:
Yeah, I agree.

Also, type in PartialChannel is a number instead of ChannelTypes. Something to update here?

Reply:
Definitely something to tackle (in a different PR)